saml-core/src/main/java/org/keycloak/saml/processing/api/util/KeyInfoTools.java

```diff
@@ -34,7 +34,7 @@
      * @param clazz
      * @return The object or {@code null} if not found.
      */
-    public static <T> T getContent(Iterable<Object> objects, Class<T> clazz) {
+    public static <T> T getContent(Iterable objects, Class<T> clazz) {
```

Comment:
<Object> is not a good idea but as a best practice, you should not remove <Object>... May be List<?> would be better

Reply:
`Iterable<?>` preferrably

Reply:
Sorry (copied/pasted from first comment)